src/terminal/Screen.zig

```diff
@@ -925,7 +925,7 @@ fn cursorScrollAboveRotate(self: *Screen) !void {
     self.clearCells(
         cur_page,
         &cur_rows[0],
```

Comment:
This needs to be changed too, no?

Reply:
Good catch. I also added some assertions to validate this more.